utility/src/main/java/org/oppia/util/parser/UrlImageParser.kt

```diff
@@ -36,11 +37,19 @@ class UrlImageParser private constructor(
   override fun getDrawable(urlString: String): Drawable {
     val imageUrl = String.format(imageDownloadUrlTemplate, entityType, entityId, urlString)
     val urlDrawable = UrlDrawable()
-    val target = BitmapTarget(urlDrawable)
-    imageLoader.load(
-      gcsPrefix + gcsResource + imageUrl,
-      target
-    )
+    if (imageUrl.endsWith("svg")) {
```

Comment:
I suggested above using a common OppiaImage so that you need just one load method in the image loader. Can't Glide identify an SVG image or do we actually need to do it?

Reply:
As we have separate implementation for loading bitmap and svg I have put a check over here.